src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -276,14 +276,21 @@ TEST(MessageHelperTestCreate,
 
   smart_objects::SmartObject& object = *smartObjectPtr;
 
+  uint64_t app_id = 1;
```

Comment:
@APipko Why do you use somewhere 64bit values and in other place 32bit? Also please don't use implicit conversions. If type is unsigned - add `u` postfix to initial value